react/features/embed-meeting/components/EmbedMeetingDialog.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getInviteURL } from '../../base/connection';
+import { Dialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { copyText } from '../../base/util';
+
+import Header from './Header';
+
+type Props = {
+
+    /**
+     * The URL of the conference.
+     */
+    url: string
+};
+
+/**
+ * Allow users to embed a jitsi meeting in an iframe.
+ *
+ * @returns {React$Element<any>}
+ */
+function EmbedMeeting({ url }: Props) {
+    /**
+     * Get the embed code for a jitsi meeting.
+     *
+     * @returns {string} The iframe embed code.
+     */
+    const getEmbedCode = () =>
+        `<iframe allow="camera; microphone; display-capture" src="${url}`
+        + 'allowfullscreen="true" style="height: 100%; width: 100%; border: 0px;"></iframe>';
+
+    /**
+     * Copy handler.
+     *
+     * @returns {void}
+     */
+    function onCopy() {
+        copyText(getEmbedCode());
+    }
+
+    return (
+        <Dialog
+            customHeader = { Header }
+            hideCancelButton = { true }
+            okKey = { 'dialog.copy' }
```

Comment:
I know we talked about what buttons to use, but when functionally testing this dialog it feels a bit lacking, because there is no feedback about the text being copied. Thus, I think it's a better idea to do something like the meeting link button, which displays a "bla bla has been copied" upon click. Can you please double check with Tibi?

Reply:
Tibi says no, I think it will stay this way until he comes up with some ux for displaying notifications within modals